### DISCUSSION TITLE
Desafio DIO

### DISCUSSION THREAD

**Discussion by Alvaro-B-Machado** [digitalinnovationone/dio-lab-open-source#48903] [Desafios DIO]
Não consigo me dar bem no desafio proposto.
[desafio](https://github.com/digitalinnovationone/dio-lab-open-source/blob/0f3d3f3c451a8c135cc0d5ded882d6dd41f494d4/CONTRIBUTING.md)

**Comment by kentonbell** [comment]
Olá! 👋

Isso normalmente acontece quando você ainda não está logado corretamente no GitHub ou quando há algum problema temporário no carregamento da página.

Tente os seguintes passos:

1. Verifique se você está logado na sua conta do GitHub.
2. Clique novamente no botão **Fork** no canto superior direito do repositório.
3. Após clicar, escolha sua conta como destino do fork.
4. Depois disso, o repositório deverá aparecer no seu perfil em **https://github.com/seu-usuario**.

Se ainda não aparecer:
- Atualize a página.
- Verifique na aba **Repositories** do seu perfil.
- Tente abrir diretamente `https://github.com/seu-usuario/dio-lab-open-source`.

Se continuar com problemas, envie um print ou descreva exatamente o que aparece na tela para que possamos ajudar melhor.
